files/bashrc.d/50-ixon

```diff
@@ -0,0 +1,3 @@
+## CTRL+S freezes the terminal (use CTRL+A to unfreeze)
```

Comment:
Isn't this c-q to unfreeze?

FWIW, I find this very useful to stop scrolling output sometimes. But I'm fine with disabling it for students.

Reply:
Ah, yeah, q. See, that’s why we need to deactivate it. I can never remember it and just press random letters with my left hand. :)

Never thought of using it to stop scrolling though, it always seemed completely mysterious to me.